Gems/Multiplayer/Code/Source/Components/NetworkCharacterComponent.cpp

```diff
@@ -107,7 +107,13 @@ namespace Multiplayer
     {
     }
 
+
     void NetworkCharacterComponent::OnActivate([[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating)
+    {
+        Physics::CharacterNotificationBus::Handler::BusConnect(GetEntityId());
+    }
+
+    void NetworkCharacterComponent::OnCharacterActivated()
```

Comment:
```
void NetworkCharacterComponent::OnPhysicsCharacterDeactivated()
{
    m_physicsCharacter = nullptr;
}
```

Reply:
I answered this in a separate code comment.